### PR TITLE
Support of `DataFrame.plot` and `DataFrame.plot.hist`

### PR DESCRIPTION
**Pull request by KathiaBarahona**
Created `PlottingHandler` class which enables access to visualization. This class should only preprocess,  execute the query and use pandas own plotting implementation. 

The functional tests use matplotlib`s testing module, so they actually compare generated images between pandas and bach

### CODE REVIEW

**Comment by jansentom** [comment]
@hendrik-obj Can you please review this?

File: bach/tests/functional/bach/test_df_plot.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+from matplotlib.testing.decorators import check_figures_equal
+from psycopg2._range import NumericRange
+
+from tests.functional.bach.test_data_and_utils import get_df_with_test_data, assert_equals_data
+
+
+# generates and compares 2 matplotlib figures (png, pdf)
+# For more information https://matplotlib.org/3.5.0/api/testing_api.html#module-matplotlib.testing
+
+@check_figures_equal(extensions=['png', 'pdf'])
```
Comment (hendrik-obj) [inline bach/tests/functional/bach/test_df_plot.py:15]: looks to me like pdf tests are skipped

File: bach/tests/functional/bach/test_df_plot.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    pbt = bt.to_pandas()
+    bins = 5
+
+    ax_ref = fig_ref.add_subplot(111)
+    ax_test = fig_test.add_subplot(111)
+    pbt.plot.hist(bins=bins, ax=ax_ref)
+    bt.plot.hist(bins=bins, ax=ax_test)
+
+    result_calc_bins = bt.plot._calculate_hist_frequencies(
+        bins=5, numeric_columns=['inhabitants'],
```
Comment (hendrik-obj) [inline bach/tests/functional/bach/test_df_plot.py:73]: `bins=bins` just to be sure?

**Review by hendrik-obj — APPROVED** [review]
lgtm, like how it's all working. see minor comments.

File: bach/tests/functional/bach/test_df_plot.py (hunk 2 above)
Comment (KathiaBarahona) [reply bach/tests/functional/bach/test_df_plot.py:73]: yes

File: bach/tests/functional/bach/test_df_plot.py (hunk 1 above)
Comment (KathiaBarahona) [reply bach/tests/functional/bach/test_df_plot.py:15]: running for me :) 
image

**Pull request merged**